core/arch/arm/plat-marvell/link.mk

```diff
@@ -0,0 +1 @@
+include core/arch/arm/kernel/link.mk
```

Comment:
minor: you can remove this file (since https://github.com/OP-TEE/optee_os/pull/1873)